apps/admin/frontend/src/api.ts

```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+import isEqual from 'lodash.isequal';
```

Comment:
I know we also use `deep-eql` for this. Might be nice to wrap one of them and export it from `@votingworks/basics` so people know where to look

Reply:
Done as 6fbd80e.